docs/docs/icicle/golang-bindings/msm.md

```diff
@@ -0,0 +1,140 @@
+# MSM
+
+
+### Supported curves
+
+`bls12-377`, `bls12-381`, `bn-254`, `bw6-761`
+
+## MSM Example
+
+```go
+package main
+
+import (
+    "github.com/stretchr/testify/assert"
```

Comment:
I would remove this and the `assert.Equal` uses in the example. These are mainly used in testing